src/main/java/com/epam/code/mie/library/services/BooksService.java

```diff
@@ -25,4 +26,9 @@ public List<Book> getAllBooks() {
     public Optional<Book> getBookByName(String name) {
         return booksRepository.findByName(name);
     }
+
+    public Book saveBook(BookDto bookDto) {
```

Comment:
Change argument to Book and remove converting logic - it will happed on facade level